src/main/java/pl/mwiski/skirmishes/SkirmishOne.java

```diff
@@ -33,6 +33,9 @@
         ints.add(0);
         ints.add(2);
         ints.add(5);
+        ints.add(6);
```

Comment:
Przez to, że wcześniej coś zrobiłeś i wypuszowałeś, to teraz w pull requeście widzę tylko te trzy zmienione linijki i niczego więcej :P